grails-app/assets/javascripts/meriplan.js

```diff
@@ -674,7 +677,7 @@ function MERIPlan(project, projectService, config) {
     };
 
     self.attachValidation = function() {
-        $('#project-details-validation').validationEngine('attach', {autoPositionUpdate:true});
+        $('#project-details-validation').validationEngine();
```

Comment:
Did you remove the autoPositionUpdate for a reason?  The MERI plan reflows a bit and it makes the validation appear on the wrong element sometimes in a full page validate (e.g. on Save).